src/chainstate/stacks/index/bits.rs

```diff
@@ -294,13 +289,26 @@ pub fn count_children(children: &[TriePtr]) -> usize {
     cnt
 }
 
+/// Read a node and its hash
+#[inline]
 pub fn read_nodetype<F: Read + Seek>(
     f: &mut F,
     ptr: &TriePtr,
 ) -> Result<(TrieNodeType, TrieHash), Error> {
     fseek(f, ptr.ptr() as u64)?;
     trace!("read_nodetype at {:?}", ptr);
-    read_nodetype_at_head(f, ptr.id())
+    read_nodetype_at_head(f, ptr.id(), true)
+}
+
+/// Read a node
+#[inline]
+pub fn read_nodetype_nohash<F: Read + Seek>(
+    f: &mut F,
+    ptr: &TriePtr,
+) -> Result<TrieNodeType, Error> {
+    fseek(f, ptr.ptr() as u64)?;
```

Comment:
Is using custom `fseek` functions much easier than just using the native method?

```suggestion
    f.seek(SeekFrom::Start(ptr.ptr() as u64))?;
```

Reply:
Lol those have been there for years.  Sure, I can remove them.

Reply:
I know, but these are new invocations, so I felt that now was as good a time as ever 😄